configs/triton_services/aws.yaml

```diff
@@ -0,0 +1,10 @@
+head_node_type:
+  name: head
+  instance_type: g5.4xlarge
+  resources:
+    cpu: 0
+worker_node_types:
+- name: 1xA10:16CPU-64GB
+  instance_type: g5.4xlarge
+  min_workers: 0
+  max_workers: 1
```

Comment:
If possible, can you use the serverless config instead? And specify `accelerator_type` in the serve deployment resource requests?